app/src/main/java/ch/epfl/favo/view/tabs/MapsPage.java

```diff
@@ -54,34 +51,45 @@
   private GpsTracker mGpsTracker;
   private ArrayList<Favor> currentActiveLocalFavorList = null;
 
+  public MapsPage() {
+    // Required empty public constructor
+  }
+
   @Override
   public void onMapReady(GoogleMap googleMap) {
     setupView();
     mMap = googleMap;
     mMap.clear();
     drawSelfLocationMarker();
     drawFavorMarker(updateFavorlist());
-    //throw new RuntimeException(
-      //  "This exception will " + "certainly be thrown out as long as Map is ready");
+    // throw new RuntimeException(
```

Comment:
is this line throwing exception still needed

Reply:
Agree, I'll remove it